Example/Tests/BPKLabelSnapshotTest.m

```diff
@@ -0,0 +1,104 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <FBSnapshotTestCase/FBSnapshotTestCase.h>
+#import <Backpack/Label.h>
+#import <Backpack/Color.h>
+
+@interface BPKLabelSnapshotTest : FBSnapshotTestCase
+
+@end
+
+NS_ASSUME_NONNULL_BEGIN
+@implementation BPKLabelSnapshotTest
+
+- (void)setUp {
+    [super setUp];
+    self.recordMode = NO;
+}
+
+- (CGRect)getDeviceFrame {
```

Comment:
Objective-c methods almost never have the `get` prefix. This method could just be called `deviceFrame` or you could just declare a constant since it's a C struct

Reply:
Ok cool, thanks. Will change.

Reply:
Check it now - does that look ok?